pkg/plugin/vm_backup_item_action.go

```diff
@@ -160,3 +172,8 @@ func volumeInDVTemplates(volume kvcore.Volume, vm *kvcore.VirtualMachine) bool {
 
 	return false
 }
+
+func isMetadataBackup(obj metav1.ObjectMeta) bool {
+	_, ok := obj.Annotations[AnnMetadataBackup]
```

Comment:
you can use k8s metav1.HasAnnotation(obj, AnnMetadataBackup) instead of this new function.
If you want to keep the function for clarity you should probably put it in util since it is being used also in the vmi_backup_item_action and though they are both under the same package not all other functions which are used in both are under util.